mkt/websites/management/commands/import_games_from_csv.py

```diff
@@ -117,19 +117,17 @@ def create_instances(self, data):
                     # Keywords use a M2M, so do that once the website is saved.
                     self.set_tags(website, row)
 
-                    WebsiteIndexer.index([website.id], no_delay=True)
-
                     # Launch task to fetch imgs once we know everything is OK.
                     try:
                         self.set_icon(website, row)
                         self.set_promo_imgs(website, row)
+                        WebsiteIndexer.index_ids([website.id], no_delay=True)
                     except Exception as e:
                         print e
+                        WebsiteIndexer.refresh_index()
```

Comment:
Yeah, mis-communication, the `refresh_index` call just forces an ES refresh to avoid potential issues due to the async nature of ES, but it does not do the indexation itself.